### DISCUSSION TITLE
More Information about unclipconditioning

### DISCUSSION THREAD

**Discussion by Creative-comfyUI** [Comfy-Org/ComfyUI#4100] [General]
It is not clear at all how strengtih and noise_augmentation is applied. Is it  applied on conditioning ?on clipvision output ? on both ? How they are mix together. With zero conditionning noise is not applied.  It wil be good to make a revision of the unclipconditioning with more parameters.

**Comment by ltdrdata** [comment]
1. The noise generated by `noise_augmentation` is indeed applied to the clip_vision_output.
2. `strength` is then applied to this entire embedding.

https://github.com/comfyanonymous/ComfyUI/blob/10c919f4c77b3615f0efa9014e8b77f294c23a2d/comfy/model_base.py#L265
